lib/801_start_server-all.sh

```diff
@@ -7,6 +7,7 @@ function start_server_init {
     DOCKER_PS="${DOCKER} ps"
     DELAY=10
     DB_NAME='swarm64da_test_db'
+    LICENSE_DOCKER_PATH='/s64da_license/s64da_license.license'
```

Comment:
I think we should reduce duplication here and just have for instance `/s64da.license` or `/s64da.lic`.

Reply:
yeah i was not sure about. Will change to /s64da_license.license , same for the toolkit and compose